src/group_theory/group_action.lean

```diff
@@ -131,20 +131,42 @@ variables {β}
 
 open quotient_group
 
+/-- Two elements `g h` of a group send `b∈β` to the same point iff they belong
+    to the same left class w.r.t the stabilizer of `b`. -/
+lemma stabilizer_coset_iff_image (b : β) (g h : α) :
+  @setoid.r _ (left_rel (stabilizer α b)) g h ↔ g • b = h • b :=
+calc _ = b ↔ g • (g⁻¹ * h) • b = g • b : (mul_action.bijective g).1.eq_iff.symm
+       ... ↔             h • b = g • b : by rw [← mul_action.mul_smul, mul_inv_cancel_left]
+       ... ↔             g • b = h • b : ⟨eq.symm, eq.symm⟩
+
+/-- Given an left coset w.r.t. the stabilizer of `b`, compute the image of `b`
+    under elements of this coset. -/
+def orbit_of_quotient_stabilizer (b : β) (x : quotient (stabilizer α b)) : orbit α b :=
+quotient.lift_on' x
+  (λ x, (⟨x • b, mem_orbit _ _⟩ : orbit α b))
+  (λ g h H, subtype.eq $ (stabilizer_coset_iff_image α b g h).1 H)
+
+/-- `orbit_of_quotient_stabilizer α b` sends `⟦x⟧` to `x • b`-/
+def orbit_of_quotient_stabilizer_spec (b : β) (x : α) :
```

Comment:
This should be a `lemma`, and probably `@[simp]`.

Reply:
Done.